flocker/acceptance/test_api.py

```diff
@@ -44,263 +38,6 @@ def can_connect():
     return dl
 
 
-class Node(PRecord):
```

Comment:
This was basically all moved into `flocker.acceptance.testtools`, with some very minor changes to `Cluster`.